src/config/i18n/en-us.json

```
@@ -1,5 +1,7 @@
 {
     "langName": "en-us",
+    "errorNisAddressNotAllowed": "You are trying to search the NEM NIS1 address in the Symbol Explorer. Please use NEM Explorer instead",
+    "errorNothingFound": "Nothing found..",
```

```suggestion
    "errorNisAddressNotAllowed": "You entered a NEM NIS1 address in the Symbol Explorer. Please use NEM Explorer instead.",
    "errorNothingFound": "Nothing found.",
```
Could the link to the NEM Explorer be given?

---

Thank you, translation improved.

> Could the link to the NEM Explorer be given?
This is a good idea but we don't have an official explorer for NIS1